plugins/kc.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kc
+spec:
+  version: v0.19.3
+  homepage: https://github.com/sunny0826/kubecm
+  shortDescription: "Easier management of kubeconfig."
```

Comment:
- no trailing period
- the description is not helpful because ALL plugins offer "easier management of something" by definition, so it would be better if there's a more specific yet still concise explanation.

Reply:
Got it, updated.
PTAL thanks.